web/src/elements/Interface/licenseSummaryProvider.ts

```diff
@@ -0,0 +1,25 @@
+import { authentikEnterpriseContext } from "@goauthentik/elements/AuthentikContexts";
```

Comment:
**3**: Make the context easily accessible to others, including a specialized accessor for the most commonly accessed field.